__mocks__/addAndRemove.js

```diff
@@ -72,11 +57,9 @@ class UserTask {
   removeTask(id) {
     this.updateId();
     const BookStored = JSON.parse(localStorage.getItem('TaskToday'));
-
     const BookFiltered = BookStored.filter((book, index) => book.index !== id);
     localStorage.setItem('TaskToday', JSON.stringify(BookFiltered));
     const testing = JSON.parse(localStorage.getItem('TaskToday'));
-    console.log(testing);
```

Comment:
Kindly remove console.log() from your project, because it's forbidden by Javascript best practice